rules/REQUEST-944-APPLICATION-ATTACK-JAVA.conf

```diff
@@ -0,0 +1,350 @@
+# ------------------------------------------------------------------------
+# OWASP ModSecurity Core Rule Set ver.3.1.0
+# Copyright (c) 2006-2016 Trustwave and contributors. All rights reserved.
+#
+# The OWASP ModSecurity Core Rule Set is distributed under
+# Apache Software License (ASL) version 2
+# Please see the enclosed LICENSE file for full details.
+# ------------------------------------------------------------------------
+
+#
+# -= Paranoia Level 0 (empty) =- (apply unconditionally)
+#
+
+
+SecRule TX:PARANOIA_LEVEL "@lt 1" "phase:1,id:944011,nolog,pass,skipAfter:END-REQUEST-944-APPLICATION-ATTACK-JAVA"
+SecRule TX:PARANOIA_LEVEL "@lt 1" "phase:2,id:944012,nolog,pass,skipAfter:END-REQUEST-944-APPLICATION-ATTACK-JAVA"
+#
+# -= Paranoia Level 1 (default) =- (apply only when tx.paranoia_level is sufficiently high: 1 or higher)
+#
+SecRule REQBODY_PROCESSOR "@rx (?:URLENCODED|MULTIPART|XML|JSON)" \
+    "id:944100,\
+    phase:1,\
+    pass,\
+    nolog,\
+    noauditlog,\
+    msg:'Enabling body inspection',\
+    tag:'paranoia-level/1',\
+    ctl:forceRequestBodyVariable=On,\
+    rev:'1',\
+    ver:'OWASP_CRS/3.1.0'"
+
+SecRequestBodyAccess On
```

Comment:
Why do we have `SecRequestBodyAccess On` here? It's in mod_security2.conf, I would remove it here. Users should generally have this enabled (or the CRS will run ineffectively anyway), but if they should disable it (maybe for debugging purposes, etc.) I feel we should honor that.

Reply:
You are right, I wrote these with an empty ruleset first so put this here to make sure it worked, I will remove this, and add a comment about it